fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -7854,6 +7855,18 @@ admin_stmt ::=
     {:
         RESULT = new AdminCancelRebalanceDiskStmt(null);
     :}
+    | KW_ADMIN KW_DECOMMISSION KW_DISK string_list:disks KW_ON STRING_LITERAL:backend
+    {:
+        RESULT = new AdminDecommissionDiskStmt(backend, disks, false);
+    :}
+    | KW_ADMIN KW_DECOMMISSION KW_DISK KW_FORCE string_list:disks KW_ON STRING_LITERAL:backend
+    {:
+        RESULT = new AdminDecommissionDiskStmt(backend, disks, true);
+    :}
+    | KW_ADMIN KW_RECOMMISSION KW_DISK string_list:disks KW_ON STRING_LITERAL:backend
+    {:
+        RESULT = new AdminRecommissionDiskStmt(backend, disks);
+    :}
```

Comment:
could u add new statment to Nerieds directly? the legacy planner will be removed in future